### DISCUSSION TITLE
🔥 Unleashing the power of Diagon to the web with Diagonjs

### DISCUSSION THREAD

**Discussion by elmouradiaminedev** [ArthurSonzogni/Diagon#71] [Show and tell]
Hey everyone! 
Exciting news – introducing [Diagonjs](https://github.com/elmouradiaminedev/diagonjs), a JavaScript wrapper for Diagon that brings its power to the web!
Now you can integrate Diagonjs into your web app, Node API, browser extension, desktop app with Electron, and more! 

Enjoy!

Amine

**Comment by ArthurSonzogni** [comment]
Excellent!
Thanks you so much! The API looks neat.

I added a section from Diagon repository to promote the new library:
https://github.com/ArthurSonzogni/Diagon/commit/c28bbc9d94de633299512f3c2f5abc9632714f89
